src/Appium.Net/Appium/AppiumDriver.cs

```diff
@@ -31,7 +31,7 @@
 namespace OpenQA.Selenium.Appium
 {
     public abstract class AppiumDriver<W> : RemoteWebDriver, IFindsById, IFindsByClassName, IFindsByName,
-        IFindsByTagName, IExecuteMethod, IFindsByFluentSelector<W>,
+        IFindsByTagName, IFindsByImage<W>, IExecuteMethod,
```

Comment:
IFindsByFluentSelector was removed deliberately?

Reply:
Yes, it is redundant because this base class implements other interfaces that inherit IFindsByFluentSelector